styles.css

```diff
@@ -1,12 +1,12 @@
 .container{
-    width: 70%;
+    width: 65%;
     position: relative;
-    left: 15%;
+    left: 14%;
 }
 .navbar{
```

Comment:
Since you have a one nav in your HTML codes, you should have just used the `nav` selector, there was no need to give it a class unless you are using this class elsewhere